### PR TITLE
allow specifying swagger documents for path converters

### PR DESCRIPTION
**Pull request by gtmanfred**
without much changing elsewhere, this allows us to set enum types for custom path converters in flask.

for example: `/v1/preferences/<preference_type:scope>` where scope could be `user`, `project`, or `account`. We can define an object with a `to_swagger` function as part of the [Flask Converter](https://flask.palletsprojects.com/en/2.3.x/api/#flask.Flask.url_map) that is defined.

### CODE REVIEW

**Review by airstandley — COMMENTED** [review]
Looks good, but would love to see a test and/or example of this in the docs.

**Comment by gtmanfred** [comment]
ok @airstandley I have gotten around to adding documentation and tests :)

File: tests/swagger_generation/registries/legacy.py [hunk 1]
```diff
@@ -5,12 +5,22 @@
 from flask_rebar import HeaderApiKeyAuthenticator
 from flask_rebar import compat
 from flask_rebar.swagger_generation import SwaggerV2Generator, SwaggerV3Generator
+from flask_rebar.swagger_generation import swagger_words as sw
 
 rebar = Rebar()
 registry = rebar.create_handler_registry()
 
+
+class UUIDPathConverter:
```
Comment (airstandley) [inline tests/swagger_generation/registries/legacy.py:14]: Awesome

**Review by airstandley — APPROVED** [review]
🎉 Looks brilliant, thanks for adding the tests and docs!

**Pull request merged**